packages/api/app/Controllers/Http/TechnologyController.js

```diff
@@ -464,13 +465,68 @@ class TechnologyController {
 		return technology;
 	}
 
-	async finalizeRegistration({ params }) {
+	async finalizeRegistration({ params, request, auth }) {
 		const technology = await Technology.findOrFail(params.id);
+		const { comment } = request.all();
+		if (comment) {
+			const user = await auth.getUser();
+			const technologyComment = await TechnologyComment.create({ comment });
+			await Promise.all([
+				technologyComment.technology().associate(technology),
+				technologyComment.user().associate(user),
+			]);
+			await technology.load('comments');
+		}
 		technology.status = technologyStatuses.PENDING;
 		await technology.save();
 		Bull.add(Job.key, technology);
 		return technology;
 	}
+
+	async sendEmailToReviewer(technology, comment = null, antl) {
+		const reviewer = await technology.getReviewer();
+		const userReviewer = await reviewer.user().first();
+		const { from } = Config.get('mail');
+		try {
+			await Mail.send(
+				'emails.changes-made',
+				{ userReviewer, technology, comment },
+				(message) => {
+					message.subject(antl('message.reviewer.changesMade'));
+					message.from(from);
+					message.to(userReviewer.email);
+				},
+			);
+		} catch (exception) {
+			// eslint-disable-next-line no-console
+			console.error(exception);
+		}
+	}
+
+	async sendToRevision({ params, request, auth }) {
+		const technology = await Technology.findOrFail(params.id);
+		const { comment } = request.all();
+		if (comment) {
+			const user = await auth.getUser();
+			const technologyComment = await TechnologyComment.create({ comment });
+			await Promise.all([
+				technologyComment.technology().associate(technology),
+				technologyComment.user().associate(user),
+			]);
+			await technology.load('comments');
+		}
+		technology.status = technologyStatuses.CHANGES_MADE;
+		await this.sendEmailToReviewer(technology, comment, request.antl);
```

Comment:
Dá pra colocar esse envio em background pra não travar o endpoint. Joga o `sendEmailToReviewer` pra um job e dá o dispatch nele aqui. Aproveita e coloca um retry lá pra caso dê erro de primeira.

Reply:
@mateus4k isso é uma ótima sugestão, porém se for fazer isso é melhor criar outra issue, visto que temos essa mesma situação em vários lugares na API.

Reply:
Criado: #560